addons/binding/org.openhab.binding.yeelight/src/main/java/org/openhab/binding/yeelight/internal/YeelightHandlerFactory.java

```diff
@@ -0,0 +1,56 @@
+/**
+ * Copyright (c) 2010-2018 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.yeelight.internal;
+
+import com.google.common.collect.Sets;
+import org.eclipse.smarthome.core.thing.Thing;
+import org.eclipse.smarthome.core.thing.ThingTypeUID;
+import org.eclipse.smarthome.core.thing.binding.BaseThingHandlerFactory;
+import org.eclipse.smarthome.core.thing.binding.ThingHandler;
+import org.openhab.binding.yeelight.handler.YeelightCeilingHandler;
+import org.openhab.binding.yeelight.handler.YeelightColorHandler;
+import org.openhab.binding.yeelight.handler.YeelightStripeHandler;
+import org.openhab.binding.yeelight.handler.YeelightWhiteHandler;
+
+import java.util.Set;
+
+import static org.openhab.binding.yeelight.YeelightBindingConstants.*;
+
+/**
+ *
```

Comment:
It would be great if you can add the missing docblocks.

Reply:
Added them for the two lacking classes. However I am not sure whether I described their purpose properly since I am not very into the project.